app/src/test/kotlin/com/waz/zclient/feature/backup/usecase/CreateBackUpUseCaseTest.kt

```
@@ -42,21 +44,20 @@ class CreateBackUpUseCaseTest : UnitTest() {
             val repo2 = mockBackUpRepo(false)
             val repo3 = mockBackUpRepo(true)
 
-            createBackUpUseCase = CreateBackUpUseCase(listOf(repo1, repo2, repo3))
+            createBackUpUseCase = CreateBackUpUseCase(listOf(repo1, repo2, repo3), testCoroutineScope)
 
             val result = createBackUpUseCase.run(Unit)
 
             verify(repo1).backUp()
             verify(repo2).backUp()
-            verifyNoInteractions(repo3)
+//            verifyNoInteractions(repo3)
```

Why the comment? If this does not make sense I would even delete it. Commented code leads to confusion and uncertainty.

---

I want to keep this comment in to remind me of the fail fast approach i need to implement, i'll add a TODO also

---

The ```TODO()``` is ok but keep this in mind. Search for all the ```TODO()``` we currently have and how old they are. 

 In my experience we never review them again. That is why my point.